testfiles/US+SF+CZ4A+hp+crawlspace+IECC_2006_VRF.idf

```diff
@@ -477,8 +477,8 @@
     0,                       !- No Cooling Supply Air Flow Rate {m3/s}
     0.595,                   !- Heating Supply Air Flow Rate {m3/s}
     0,                       !- No Heating Supply Air Flow Rate {m3/s}
-    autosize,                !- Cooling Outdoor Air Flow Rate {m3/s}
-    autosize,                !- Heating Outdoor Air Flow Rate {m3/s}
+    0,                !- Cooling Outdoor Air Flow Rate {m3/s}
+    0,                !- Heating Outdoor Air Flow Rate {m3/s}
```

Comment:
Does including OA flow cause a difference between the sum of TU heating capacity + piping losses and condenser total heating capacity?

Reply:
it does cause some more difference

Reply:
@rraustad Here is the output file when the heating cooling air flow rate is autosize
[eplusout_when heating cooling air flow rate autosize.xlsx](https://github.com/user-attachments/files/16786706/eplusout_when.heating.cooling.air.flow.rate.autosize.xlsx)

The following is a snapshot sorted with column O in descending order

![image](https://github.com/user-attachments/assets/c3cfa1e2-fab9-46da-8192-7130ea837e70)

Reply:
If L x M = 9083.83288 * 0.98793126 = 8974.2, which closely matches column K at 8973.19993, then why is the TU heating coil "allowed" to provide more than that "max" heating rate? Is the TU heating coil capacity getting limited by the MaxHeatingCapacity variable? I recall discussing that function LimitTUCapacity should include piping losses.

Reply:
Coil capacity should have been limited. Maybe there's some lingering issues there. I will check on that.